dlgr/griduniverse/game_config.yml

```diff
@@ -288,6 +288,26 @@ items:
     item_count: 0
     sprite: "emoji:🌴"
 
+  - name: Stag
+    item_id: stag
+    n_uses: 1
+    sprite: "emoji:🦌"
+    spawn_rate: 0.05
+    item_count: 5
+    portable: false
+    crossable: true
+    interactive: true
+
+  - name: Fallen Stag
+    item_id: fallen_stag
+    n_uses: 1
+    sprite: "emoji:👜"
+    interactive: false
```

Comment:
It doesn't really matter now that @jessesnyder has fixed the client side bug in handling zero-calorie non-interactive items, but I think this should probably be `interactive: true`, since the fallen stag is likely to be a useful item and is not immediately edible. I believe that `interactive: false` is primarily intended for Pac Man™ style legacy food-type items.